code/modules/instruments/objs/items/instruments.dm

```diff
@@ -153,22 +167,22 @@
 	lefthand_file = 'icons/mob/inhands/items_lefthand.dmi'
 	righthand_file = 'icons/mob/inhands/items_righthand.dmi'
 	attack_verb = list("beautifully honks")
-	instrumentId = "bikehorn"
-	instrumentExt = "ogg"
 	w_class = WEIGHT_CLASS_TINY
 	force = 0
 	throw_speed = 3
-	throw_range = 7
+	throw_range = 15
```

Comment:
Why the increase here?

Reply:
Seems to have been overwritten by [TG's throw range](https://github.com/tgstation/tgstation/blob/1aa293ea337283a0191140a878eeba319221e5df/code/game/objects/items/devices/instruments.dm#L289), I'll fix it